cfgov/v1/atomic_elements/molecules.py

```diff
@@ -83,6 +83,15 @@ class AbstractHero(blocks.StructBlock):
             "</ul>"
         ),
     )
+
+    heading_with_line_break = blocks.CharBlock(
+        required=False,
+        help_text=mark_safe(
+            "Use if the heading needs to break to a second line"
+            "at a specific point in the text."
+        ),
```

Comment:
While I do see some other help text in this module uses it to do some HTML formatting with the help text, [`mark_safe`](https://docs.djangoproject.com/en/5.0/ref/utils/#django.utils.safestring.mark_safe) looks unnecessary here.

```suggestion
        help_text=(
            "Use if the heading needs to break to a second line"
            "at a specific point in the text."
        ),
```